lib/maze/checks/assert_check.rb

```diff
@@ -39,6 +39,20 @@ def operator(operand1, operator, operand2, message = nil)
       def kind_of(klass, object, message = nil)
         assert_kind_of(klass, object, message = nil)
       end
+
+      def block(message = 'block failed', &block)
+        assert_block(message, &block)
+      end
+
+      def include(collection, object, message = nil)
+        assert_include(collection, object, message)
+      end
+      alias includes include
+
+      def refute_include(collection, object, message = nil)
+        Minitest::Assertions.refute_includes(collection, object, message)
```

Comment:
[`refute_includes` does exist in Test Unit v3+](https://ruby-doc.org/stdlib-3.0.0/libdoc/test-unit/rdoc/Test/Unit/Assertions.html#method-i-refute_includes), we might just need to update?

If we do that then we may be able to remove Minitest as a dependency of Maze Runner entirely

Reply:
It might also make sense to change the step that uses this to not use it, if it's just here for one or two tests

Reply:
I certainly see some benefit in removing the minitest dependency if we can.  I'm slightly worried about breaking pipelines by removing the dependency, but that wouldn't be too hard to mop up.

I'd also be very happy to see `assert_not_include` used rather than a refute.

Reply:
👍 `assert_not_includes` sounds better to me too and is more consistent with `assert_nil`/`assert_not_nil`